modules/machine-user-infra-machines-iso.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
+
+[id="machine-user-infra-machines-iso_{context}"]
+= Creating more {op-system-first} machines using an ISO image
+
+You can create more compute machines for your bare metal cluster by using an
+ISO image to create the machines.
+
+.Prerequisites
+
+* Obtain the URL of the Ignition config file for the compute machines for your
+cluster. You uploaded this file to your HTTP server during installation.
+* Obtain the URL of the BIOS or UEFI {op-system} image file that you uploaded
+to your HTTP server during cluster installation.
+
+.Procedure
+
+. Use the ISO file to install {op-sytem} on more compute machines. Use the same
+method that use used when you created machines before you installed the cluster:
+** Burn the ISO image to a disk and boot it directly.
+** Use ISO redirection via a LOM interface.
+
+. After the instance boots, press the `TAB` or `E` key to edit the kernel command line.
+. Add the parameters to the kernel command line:
++
+----
+coreos.inst=yes
+coreos.inst.install_dev=sda <1>
+coreos.inst.image_url=<bare_metal_image_URL> <2>
+coreos.inst.ignition_url=http://example.com/config.ign <3>
```

Comment:
Change config.ign to worker.ign, this is the name used by openshift-install to generate file

Reply:
@morenod, is this true only when you add more compute/worker machines, or is it also true during cluster initialization?

Reply:
always you install a worker, you need the worker.ign file